ctapipe/instrument/camera.py

```diff
@@ -518,6 +533,69 @@ def get_shower_coordinates(self, x, y, psi):
 
         return longi, trans
 
+    def get_pixel_id(self, x, y):
```

Comment:
I'd suggest renaming this - it's not clear what the input is ("get" could mean just lookup the pixel_id from the pixel *index*). Maybe `position_to_pixel_id(x,y)`?

Reply:
renamed method to position_to_pix_index() in 49e3572.